src/main/java/UserInterface.java

```diff
@@ -0,0 +1,101 @@
+import java.util.Scanner;
+
+public class UserInterface {
```

Comment:
Good implementaiton of UI. Possibly implement standardized printing method

Reply:
I did have that on my TODO list, might get around to it in future commits!